jobs/kpi-forecasting/kpi_forecasting/configs/search_forecasting_daily_active_users.yaml

```diff
@@ -0,0 +1,51 @@
+---
+metric_hub:
+  app_name: "multi_product"
+  slug: "search_forecasting_daily_active_users"
+  alias: "search_forecasting_daily_active_users"
+  start_date: "2018-01-01"
+  end_date: NULL
+
+forecast_model:
+  model_type: "prophet"
+  segment_settings:
```

Comment:
What do the values under "segment_settings" mean?  Should they be unpacked as separate models or does Prophet expect this and know what to do with it?  It looks like you might need to iterate through it in kpi_forecasting.py

Reply:
Yeah, it'll be unpacked in the forecast class, depending on which segment is being run -- by segment, I mean if the model is being trained to forecast either mobile or desktop metrics.